package/network/config/ltq-vdsl-app/files/10_atm_ptm.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+[ "$DSL_NOTIFICATION_TYPE" = "DSL_STATUS" ] || exit 0
+
+if [ "$DSL_TC_LAYER_STATUS" = "ATM" ]; then
+
+	if grep -q "ltq_ptm_vr9" /proc/modules ; then
+		# TODO: make ltq_atm_vr9/ltq_ptm_vr9 "hot-swappable"
+		# ifconfig ptm0 down &>/dev/null
+		# rmmod "ltq_ptm_vr9"
+		logger -p daemon.notice -t "dsl-notify" "Switching to TC-Layer ATM while PTM driver is loaded. Rebooting System"
```

Comment:
That's interesting, in the Netgear DM200 port we had to strap two gpio pin states depending on ATM/PTM configuration.